js/util/combineCriteria.ts

```diff
@@ -0,0 +1,31 @@
+import * as Collections from 'typescript-collections';
+import { PropExtractorCriteria, PropExtractorConfig } from './extractProps';
+
+export const getCombinedCriteria = (
+  componentName: string,
+  configs: PropExtractorConfig[]
+): PropExtractorCriteria => {
```

Comment:
Since this is really just producing a list of props to include (rather than a list to include and/or exclude) this should probably just return the inclusion array.

Reply:
Yeah - I like that.  Refactored with the last commit.